schema.graphql

```diff
@@ -1,8 +1,19 @@
-type Proposal @entity {
-  id: ID!
-  address: Bytes!
-  numOfChoices: BigInt!
-  organization: Bytes!
-  paramsHash: Bytes!
-  proposer: Bytes!
+type ReputationHolder @entity {
```

Comment:
should we have schema.graphql under directories (e.g /reputation ?)

Reply:
I thought about this, yeah I think we probably want to modularize the graphql schema. This can easily be done via [handlebars partials](https://handlebarsjs.com/partials.html)

Reply:
Implemented in: https://github.com/daostack/subgraph/pull/6/commits/f79f0d2650de81c95150b4000992ca90ec26c5e0 What do you think?

Reply:
hmm.. yeah can be removed.